src/gui/dialogs/statistics_dialog.cpp

```diff
@@ -275,14 +277,23 @@ void statistics_dialog::add_hits_row(
 	std::map<std::string, string_map> data;
 	string_map item;
 
+	std::pair<std::string, std::string> pair;
+
 	item["label"] = type;
 	data.emplace("hits_type", item);
-	item["label"] = tally(by_cth);
+
+	pair = tally(by_cth);
+	item["label"] = pair.first;
 	data.emplace("hits_overall", item);
+	item["label"] = pair.second + spacer;
```

Comment:
Likewise, make the spacer a part of the string so that you don't need to append it afterwards.